Ginger/GingerCore/Actions/UIAutomation/UIAComWrapperHelper.cs

```diff
@@ -3323,6 +3321,40 @@ public override string SetControlValue(object obj, string value)
                 throw new Exception("Unable to set value. Value - " + value);
             }
         }
+
+        /// <summary>
+        /// This method is used to select the
+        /// </summary>
+        /// <param name="element"></param>
+        /// <param name="val"></param>
+        private void SetCombobValueByUIA(AutomationElement element, string val)
+        {
+            try
+            {
+                ExpandCollapsePattern exPat = element.GetCurrentPattern(ExpandCollapsePattern.Pattern)
+                                                                              as ExpandCollapsePattern;
+
+                if (exPat == null)
+                {
+                    throw new ApplicationException("Unable to set value");
+                }
+
+                exPat.Expand();
+
+                AutomationElement itemToSelect = element.FindFirst(TreeScope.Descendants, new
+                                      PropertyCondition(AutomationElement.NameProperty, val));
+
+                SelectionItemPattern sPat = itemToSelect.GetCurrentPattern(
+                                                          SelectionItemPattern.Pattern) as SelectionItemPattern;
+                sPat.Select();
+            }
+            catch (Exception e)
+            {
+                Reporter.ToLog(eLogLevel.DEBUG, "In Combo Box Exception vp is null::");
+                throw new Exception("Element doesn't support ValuePattern.Pattern, make sure locator is finding the correct element");
```

Comment:
log info you writes in catch block are still valid although you added this handling?

Reply:
yes the info is required, if this doesn't works at some different scenario